metabrainz/templates/users/supporters-list.html

```diff
@@ -24,7 +24,7 @@ <h1 class="page-title">Supporters</h1>
         <div class="users-grid">
           {% for user in featured_users %}
             <div class="user">
-              <a href="{{ url_for('users.tier', tier_id=tier.id) }}">
+              {% if user.website_url %}<a href="{{ user.website_url }}">{% endif %}
```

Comment:
We are currently not showing there which users have bad standing. Not sure if we should do it there like on tier page (see my change to *metabrainz/templates/users/tier.html* in this pull request).